model/lifecycle.go

```diff
@@ -476,56 +476,74 @@ func AddTasksToBuild(b *build.Build, project *Project, v *version.Version,
 	return b, nil
 }
 
+// BuildCreateArgs is the set of parameters used in CreateBuildFromVersion
+type BuildCreateArgs struct {
+	Project      *Project         // project to create the build for
+	Version      *version.Version // the version the build belong to
```

Comment:
should these be not-pointers? what does it mean if they are pointers.

Reply:
Made them non pointers since the function expects them to be always populated